logfire/_internal/main.py

```diff
@@ -1698,7 +1698,17 @@ def message_template(self) -> str | None:  # pragma: no cover
 
     @property
     def tags(self) -> Sequence[str]:  # pragma: no cover
```

Comment:
```suggestion
    def tags(self) -> tuple[str, ...]:
```

Please test this case.